test/passing/tests/functor.ml

```diff
@@ -110,3 +110,11 @@ module M : functor (A : S) -> functor (B : S) -> S = N
 module M : functor (A : S) (B : S) -> S = N
 module M : functor (A : S) -> functor (B : S) -> S = N
 module M : (A : S) -> functor (B : S) -> S = N
+
+[@@@ocamlformat "break-struct=natural"]
+
+module M = F ((struct type t end : sig type t end))
+
+module M = struct type t end
```

Comment:
Are the tests testing anything ?

Reply:
Yes, line 116 would be broken in multiple lines before this PR. line 118 is shere to showcase the inconsistency when that happens.

Reply:
I see!